src/styles/user-signup.css

```diff
@@ -0,0 +1,66 @@
+.SignUpForm {
```

Comment:
Run prettier over your work, all these classes should be on the far left of the file. Ill tag you in a post Dan put on slack on how to do this, will make your life easier!